### DISCUSSION TITLE
Cheating is bad

### DISCUSSION THREAD

**Discussion by kuznetsovgm** [shamhi/HamsterKombatBot#2363] [General]
Кто уже получил это достижение?

**Comment by libx264** [comment]
Я получил, но только на одном, более прокачанном акке.

**Comment by lolyinseo** [comment]
Ключи от Minigame юзал после изменения подписи ?

**Comment by princeofscale** [comment]
Да вроде похуй, не забанили же

**Comment by lunaticsm** [comment]
everyones got that achievement, the difference only a grey one and the colorful one.

**Comment by lunaticsm** [comment]
> Да вроде похуй, не забанили же

they don't ban you, they just decrease something when they on the market

**Comment by Afkaros** [comment]
Всем за генерацию ключей дали?
